### ISSUE TITLE
Creating a new Widget doesn't have Key as an nullable parameter

### ISSUE THREAD

**Issue by Tinhorn** [marius-h/flutter_enhancement_suite#231]
**Describe the bug**

If you create a new widget through the new menu, you will get a widget that has the key parameter not null
**Steps to Reproduce**

Go to New | Flutter Widget.
Create any file.
The key parameter is named but not nullable

```dart
class Foo extends StatelessWidget {
  const Foo({Key key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return Container();
  }
}
```

**Expected behavior**

```dart
class Foo extends StatelessWidget {
  const Foo({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return Container();
  }
}
```

**Screenshots**

**Software used**

```
IntelliJ IDEA 2019.2 (Ultimate Edition)
Build #IU-192.5728.98, built on July 23, 2019
Runtime version: 11.0.3+12-b304.10 amd64
Windows 10 10.0
```

**Additional context**

**Comment by Wiesmak** [comment]
I think you meant this as the expected one:
```dart
class Foo extends StatelessWidget {
const Foo({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
  return Container();
  }
}
```
Still, this should be fixed ASAP as it requires little work and adding those `?` every time is quite frustrating.

**Comment by Tinhorn** [comment]
> I think you meant this as the expected one:
> 
> ```dart
> class Foo extends StatelessWidget {
> const Foo({Key? key}) : super(key: key);
> 
>   @override
>   Widget build(BuildContext context) {
>   return Container();
>   }
> }
> ```
> 
> Still, this should be fixed ASAP as it requires little work and adding those `?` every time is quite frustrating.

Fixed it. Thanks.

**Comment by Wiesmak** [comment]
protip:
you add a code block by typing
` ```language `
\<YOUR CODE\>
` ``` `

**Comment by Wiesmak** [comment]
also, seems this got fixed in #173, but for some reason the intellij has still an old version

**Comment by Tinhorn** [comment]
@Wiesmak do you have the same issue?
This might be an installation on my end.
Thank you for your time btw

**Comment by Wiesmak** [comment]
yeah, it happens all the time

**Comment by marius-h** [comment]
Fixed in v1.6.1 ✅
